### ISSUE TITLE
Settings menu has trouble staying on top.

### ISSUE THREAD

**Issue by CharlesCorrin** [mgba-emu/mgba#1425]
If you go into settings and use a drop-down menu from within to change a setting, but then don't actually change the setting, the emulator will push the settings window to the back, behind the main window. This only happens once per opening of the setting window.

Quick test: Open settings. Open the audio driver's drop-down menu.  Choose the same option that was already selected. Is SDL for me. The emulator will force the settings menu to the back. Bring the settings back to the foreground and try again. It won't do it now. But close and re-open the settings and it can now happen again.

Using the latest official build as of this writing. This is a regression from some point, because it didn't always do this.

**Comment by endrift** [comment]
I can't reproduce. What OS are you on (and if Linux,what windowing system are you using)?

**Comment by CharlesCorrin** [comment]
Sorry. I hadn't considered that.

I'm using Windows 10 Home 64-bit Version 1809.

**Comment by CharlesCorrin** [comment]
Oops. I found something. My report was actually incorrect. It only does it if you click the same location twice. So you click to open the drop-down menu, then you click on the same spot again. Just double-click on it.

**Comment by endrift** [comment]
Still can't reproduce on Linux. Will try Windows later.

**Comment by endrift** [comment]
Can't reproduce on Windows.

**Comment by CharlesCorrin** [comment]
Oh. That's a shame. It's really easy for me to do by accident (and on purpose). The bug is so irritating, so I was hoping it would be fixed. Maybe it can be fixed in in the future, even if by accident. :(

**Comment by CharlesCorrin** [comment]
I decided to go ahead and do it now. And I succeeded. I confirmed that the last Windows build without the bug was:
https://github.com/mgba-emu/mgba/commit/f8cd425fad34dca37c373cee762915202a07e686

I can confirm the bug does occur on:
https://github.com/mgba-emu/mgba/commit/6ee24a21de74fd76fe0af51ee4db7db2e6fcc786

There are two more commits between those two, but they didn't have Windows builds for me to test. I'll list those commits below in case it helps:
https://github.com/mgba-emu/mgba/commit/27f8abff155aad5df5632b415a8d83213586a34f
https://github.com/mgba-emu/mgba/commit/3c5a9258a72ef9b7f5904783857c42232064119d
